### PR TITLE
Complete italian translation for the wires update

### PR DESCRIPTION
**Pull request by Doc-Ninja**
Should be typo-free

### CODE REVIEW

**Comment by Doc-Ninja** [comment]
It fails the Yaml lint check but it's not because of my file, should be complete and ready to merge

File: translations/base-it.yaml [hunk 1]
```diff
@@ -4,49 +4,49 @@ steamPage:
         infinita.
     discordLinkShort: Official Discord
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:5]: Not translated? Should be `Ufficiale` or something else like in Steam description

File: translations/base-it.yaml [hunk 2]
```diff
@@ -344,41 +345,41 @@ ingame:
...
+        one_miner: 1 trivella
+        n_miners: <amount> trivelle
+        limited_items: Limitato a <max_throughput>
     watermark:
-        title: Demo version
-        desc: Click here to see the Steam version advantages!
-        get_on_steam: Get on steam
+        title: Versione demo
+        desc: Clicca qui per vedere i vantaggi della versione Steam!
+        get_on_steam: Ottieni su steam
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:354]: `steam` upper or lowercase? In Steam description it's uppercase

File: translations/base-it.yaml [hunk 3]
```diff
@@ -672,74 +672,74 @@ storyRewards:
...
             completa!
     reward_stacker:
-        title: Stacker
-        desc: You can now combine shapes with the <strong>stacker</strong>! Both inputs
-            are combined, and if they can be put next to each other, they will
-            be <strong>fused</strong>. If not, the right input is
-            <strong>stacked on top</strong> of the left input!
+        title: Impilatrice
+        desc: Ora puoi combinare forme con <strong>l'impilatrice</strong>! I due ingressi
+                vengono combinati e se possono essere messi uno accanto all'altro, verranno
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:677]: You tab in one too much in this and the following two

File: translations/base-it.yaml [hunk 4]
```diff
@@ -468,125 +469,124 @@ buildings:
...
-                too.
+            name: Filtro
+            description: Collega un segnale per mandare tutti gli oggetti corrispondenti verso l'alto e
+                i restanti verso destra. Può essere controllato anche con segnali booleani.
     display:
         default:
             name: Display
-            description: Connect a signal to show it on the display - It can be a shape,
-                color or boolean.
+            description: Collega un segnale per mostrarlo sul display.
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:550]: `Display` upper or lower case? compare to line 701

Comment (Doc-Ninja) [reply translations/base-it.yaml:550]: it's consisted with english, I can change it either way

Comment (DerFeldspatz) [reply translations/base-it.yaml:550]: I don't think it is intended. Just change it how nouns are written in Italian

Comment (DerFeldspatz) [reply translations/base-it.yaml:550]: But consistently through the whole text if possible 😅

Comment (Doc-Ninja) [reply translations/base-it.yaml:550]: well display is a common name so lowercase

Comment (Doc-Ninja) [reply translations/base-it.yaml:550]: i think those are the only two instances
i'll do a final pass on everything anyway

**Comment by Doc-Ninja** [comment]
Done the final pass, should be ok

**Comment by tobspr** [comment]
Awesome, thank you! :)

**Pull request merged**